### ISSUE TITLE
`gr status` column headers and long branch names

### ISSUE THREAD

**Issue by shousper** [mixu/gr#77]
Hi, long time user, first time bug/feature report.

The recent additions to include the current branch name and remote status information have kind of ruined the `gr status` output for me.
Our branches are usually anywhere from 20-40 characters long. Sadly this means the output looks something like this:

```
~/Source/project-a          feature/ABCD-17000-banana 3 modified   [ahead 1, behind 4]@app
~/Source/project-b          master         Clean                     @lib
```

Could the output here automatically line things up in a more table-like layout, maybe it's also time to add some column headings?
The branch names could even be intelligently truncated if they're too long on a terminal window that isn't wide enough, likewise for the ahead/behind output.
If this stuff was configurable in the `.grconfig` it could make it easier to release the feature without forced adoption too ;)

Let me know what you think, cheers.

**Comment by mixu** [comment]
yeah I think making these configurable seems reasonable. I don't have the bandwidth to make that happen soon, but if you have a bit of time I'd be happy to accept a PR!
